### PR TITLE
Bump sentry version and enable performance metrics

### PR DESCRIPTION
**Pull request by Fatxx**
**Development & PR Process**
1. Follow MetaMask [Mobile Coding Standards](https://docs.google.com/document/d/1VJLwTRsUw_5EDq_o8d6sSbXUAYENLurkRitYO45eY5o/edit?usp=sharing)
2. Add `release-xx` label to identify the PR slated for a upcoming release (will be used in release discussion)
3. Add `needs-dev-review` label when work is completed
4. Add `needs-qa` label when dev review is completed
5. Add `QA Passed` label when QA has signed off

**Description**

This PR aims to add support for new Sentry Performance monitoring metrics.

Gotchas:
 - The screenshot presented below is from a personal project since we are already on the limit of transactions in our Sentry I had to take this route in order to test the functionality;
 - Migration of native code doesn't require any changes on our codebase;

**Screenshots/Recordings**
Screen Shot 2023-03-22 at 09 22 28
**Issue**

Progresses #???

**Checklist**

* [ ] There is a related GitHub issue
* [ ] Tests are included if applicable
* [ ] Any added code is fully documented

### CODE REVIEW

**Comment by Fatxx** [comment]
@SocketSecurity @sentry/cli@1.75.0

File: app/util/sentryUtils.js [hunk 1]
```diff
@@ -174,7 +174,7 @@ export function setupSentry() {
               }),
             ]
           : integrations,
-      tracesSampleRate: 0.2,
+      tracesSampleRate: 0.05,
```
Comment (tommasini) [inline app/util/sentryUtils.js:177]: Just curiosity, why are we just wanting a sample rate of 5% of transactions?

**Review by tommasini — APPROVED** [review]
LGTM! Just please look at what is failing on the pipeline

File: app/util/sentryUtils.js (hunk 1 above)
Comment (Fatxx) [reply app/util/sentryUtils.js:177]: To control our Sentry limits but this will change

**Comment by Fatxx** [comment]
This needs further change in sample rate

**Review by NicolasMassart — COMMENTED** [review]
C: prevent including unrelated formatting changes in the PR.
The wdio folder is excluded from linter so this will not make your PR fail if you don't apply prettier to it.

**Comment by sethkfman** [comment]
@SocketSecurity ignore @sentry/cli@1.75.0

**Review by sethkfman — APPROVED** [review]
LGTM, fix the yarn.lock conflicts and it can be merged

**Pull request merged**